server/FakeToOriginalIPConverter/FakeToOriginalIPConverter.js

```diff
@@ -92,7 +92,7 @@ async function modifyServer() {
             let application = applicationList[index];
             let applicationName = application['component'];
             let releaseNumber = application['release'];
-            let originalTcpIpAddress = application['original-address'];
```

Comment:
The functions modifyNewRelease() and modifyOldRelease() can be made based on FC 
OldRelease = PromptForEmbeddingCausesRequestForBequeathingData and 
NewRelease = PromptForBequeathingDataCausesNewApplicationBeingRequestedToInquireForApplicationTypeApprovals

Reply:
To be discussed in team before deciding

Reply:
currently, initial configurations will have application-name = OldRelease and NewRelease, so no changes necessary in code for now. In future, if needed, we can update the logic